### PR TITLE
Refactor/aws integration

### PR DESCRIPTION
**Pull request by Manjish**
- Added AWS Services
- Added Cognito Auth Middleware
- Added Admin Seed to Cognito
- Changed upload middleware from Google Bucket to AWS S3
- Added Amazon SES

### **API Information**

| Endpoint     | Method | Description             |
| ------------ | ------ | ----------------------- |
| `/api/upload-test` | POST    | Uploads the image to Amazon S3 and returns presigned URL |
| `/api/send-test-email` | POST    | Sends email with Subject and Body to the provided email from Amazon SES |

### CODE REVIEW

File: domain/user/module.go
Comment (Denes-cilwal) [inline domain/user/module.go:1]: @Manjish Let's remove this as auto migrate  invocation and stick to writing migration for schema.

File: domain/user/repository.go [hunk 1]
```diff
@@ -19,6 +19,7 @@ func NewRepository(db infrastructure.Database, logger framework.Logger) Reposito
 	return Repository{db, logger}
 }
 
+// For AutoMigrating (used in fx.Invoke)
 func Migrate(r Repository) error {
 	r.logger.Info("[Migrating...User]")
 	if err := r.DB.AutoMigrate(&models.User{}); err != nil {
```
Comment (Denes-cilwal) [inline domain/user/repository.go]: Let revert these changes ! @Manjish

**Review by Denes-cilwal — CHANGES_REQUESTED** [review]
@Manjish  Let remove this cognito and upload file implementation. Adding up only and services is ok 🙏🏾

File: domain/user/repository.go (hunk 1 above)
Comment (paudelgaurav) [reply domain/user/repository.go]: will be fixed by https://github.com/wesionaryTEAM/go_clean_architecture/pull/114

File: domain/user/module.go
Comment (paudelgaurav) [reply domain/user/module.go:1]: will be fixed by https://github.com/wesionaryTEAM/go_clean_architecture/pull/114

**Comment by paudelgaurav** [comment]
Please merge https://github.com/wesionaryTEAM/go_clean_architecture/pull/114 before merging this PR

**Review by Denes-cilwal — APPROVED** [review]
LGTM 🚀
